src/refract/JSONSchemaVisitor.cc

```diff
@@ -340,62 +336,22 @@ namespace refract
 
     void JSONSchemaVisitor::visit(const NullElement& e)
     {
-        NullElement nil;
-        addMember("type", &nil);
+        addMember("type", new NullElement);
```

Comment:
Is this heap allocation going to be cleaned up somewhere? I would prefer the original code.